L/libsasl2/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libsasl2"
+version = v"2.1.28"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-$(version)/cyrus-sasl-$(version).tar.gz",
+                  "3e38933a30b9ce183a5488b4f6a5937a702549cde0d3287903d80968ad4ec341")
```

Comment:
I even computed the new hash locally, but forgot to copy it here
```suggestion
                  "7ccfc6abd01ed67c1a0924b353e526f1b766b21f42d4562ee635a8ebfc5bb38c")
```

Reply:
```
[ Info: Downloading https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz to /cache/yggdrasil/downloads/3e38933a30b9ce183a5488b4f6a5937a702549cde0d3287903d80968ad4ec341-cyrus-sasl-2.1.28.tar.gz...
┌ Error: Hash Mismatch!
│   Expected sha256:   3e38933a30b9ce183a5488b4f6a5937a702549cde0d3287903d80968ad4ec341
│   Calculated sha256: 7ccfc6abd01ed67c1a0924b353e526f1b766b21f42d4562ee635a8ebfc5bb38c
└ @ BinaryBuilderBase /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilderBase/HbzWv/src/ArchiveUtils.jl:113
```
What the hell is going on now